completions/bash/podman

```diff
@@ -2163,6 +2163,7 @@ _podman_podman() {
            --storage-driver
            --storage-opt
            --log-level
+           --namespace
```

Comment:
Are there plans for a "getnamespace" command to list all the of the known namespaces similar to "kubectl get namespaces"?

Reply:
Right now we don't actually keep track of them at all, aside from associating them with pods/containers. We could certainly add this if there's demand, but my current attitude is wait-and-see

Reply:
Right I don't really see this as something people will use.  I see this as CRI-O having a namepsace and podman just using "" so it will see all pod/containers, while CRI-O only sees its pods and containers.

Reply:
I think it would be nice to have a libpod API *and* a "pod"man command for that. On one hand, the API would be more complete, on the other, podman would be more useful for debugging. I wouldn't like to switch between podman, crictl and kubectl to shuffle data around. IMHO that's no blocker at all for this PR but a nice-to-have.